lib/bazel/options.go

```diff
@@ -62,6 +69,11 @@ func (o *queryOptions) Args() []string {
 	if o.unorderedOutput {
 		f = append(f, "--order_output=no")
 	}
+	if o.workspaceLog != nil {
```

Comment:
it looks like the workspace log file is required, maybe just error out if it wasn't passed?

Reply:
Shouldn't be - the workspace log is only parsed if the flag is passed?

Reply:
See below review